protocol/migrations/sqlite/1664452901_add_discord_messages_and_authors_indices.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX idx_dm_author_id_dm_id ON discord_messages (id, author_id);
+CREATE INDEX idx_response_to_source_discord_message_id ON user_messages (response_to, source, discord_message_id);
+CREATE INDEX idx_pinned_response_to_source_discord_message_id ON pin_messages (message_id, pinned);
```

Comment:
I tried adding these to improve query performance.
Turns out with discord community with 200000+ messages, message querying can get slower.